org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -915,6 +915,56 @@ private void TransformTarget()
                         // }
 
+                        if (!transformUpdated.IsMaskSet(TransformFlags.Move))
+                        {
+                            Target.transform.position = smoothingActive ?
+                                Smoothing.SmoothTo(Target.transform.position, constraintTranslate.Position, translateLerpTime, Time.deltaTime) :
+                                constraintTranslate.Position;
+                        }
+                    }
+                    else if (currentHandle.HandleType == HandleType.Translation2D)
+                    {
+                        Vector3 translateVectorOnPlane = Vector3.ProjectOnPlane(currentGrabPoint - initialGrabPoint, currentHandle.transform.forward);
+
+                        var goal = initialTransformOnGrabStart.Position + translateVectorOnPlane;
+                        MixedRealityTransform constraintTranslate = MixedRealityTransform.NewTranslate(goal);
+                        if (EnableConstraints && constraintsManager != null)
+                        {
+                            constraintsManager.ApplyTranslationConstraints(ref constraintTranslate, true, currentHandle.IsGrabSelected);
+                        }
+
+                        // TODO: Elastics integration (soon!)
+
+                        // if (elasticsManager != null)
+                        // {
+                        //     transformUpdated = elasticsManager.ApplyTargetTransform(constraintTranslate, TransformFlags.Move);
+                        // }
+
+                        if (!transformUpdated.IsMaskSet(TransformFlags.Move))
+                        {
+                            Target.transform.position = smoothingActive ?
+                                Smoothing.SmoothTo(Target.transform.position, constraintTranslate.Position, translateLerpTime, Time.deltaTime) :
+                                constraintTranslate.Position;
+                        }
+                    }
+                    else if (currentHandle.HandleType == HandleType.Translation3D)
+                    {
+                        Vector3 translateVector = currentGrabPoint - initialGrabPoint;
+
+                        var goal = initialTransformOnGrabStart.Position + translateVector;
+                        MixedRealityTransform constraintTranslate = MixedRealityTransform.NewTranslate(goal);
+                        if (EnableConstraints && constraintsManager != null)
+                        {
+                            constraintsManager.ApplyTranslationConstraints(ref constraintTranslate, true, currentHandle.IsGrabSelected);
+                        }
+
+                        // TODO: Elastics integration (soon!)
+
+                        // if (elasticsManager != null)
+                        // {
+                        //     transformUpdated = elasticsManager.ApplyTargetTransform(constraintTranslate, TransformFlags.Move);
+                        // }
```

Comment:
Thank you for your contribution, this change looks great. We are trying to keep the codebase as clean as possible so we encourage contributors to avoid leaving commented code out of main branch. Could you please remove the commented code?

Reply:
I understand your point of view, I too leave commented sections in my personal branches.  However, since this is an open-source project I think it is best to leave the code as clean as possible because not everyone in the community may have the same context as the author.  @MixedRealityToolkit/mixedrealitytoolkit-unity-maintainers What are your thoughts on this?